__tests__/AdminLogin.tsx

```diff
@@ -21,13 +21,9 @@ import {
 
 describe('AdminLogin', () => {
   it('performs the admin login flow via OAuth2', async () => {
-    (getOrganizationList as jest.Mock).mockReturnValue(() =>
```

Comment:
thx!.. for some reason the other way didn't work for me but it was probably due to something unrelated.

Reply:
It didn't work before because client functions were higher order functions (they returned other functions). Now they're regular async functions, so `mockResolvedValue()` works